packages/react-vapor/src/components/select/SelectConnected.tsx

```diff
@@ -154,7 +151,7 @@ export class SelectConnected extends React.PureComponent<ISelectProps> {
                     });
                 }
             });
-            return <div className="flex p2 flex-center bg-white flex-column mod-border-bottom">{newChildren}</div>;
+            return <div className="flex p2 flex-center flex-column mod-border-bottom">{newChildren}</div>;

```

Comment:
The background is now transparent, which causes issues.

![image](https://user-images.githubusercontent.com/35579930/106476761-6564fb00-6475-11eb-9537-831f900a19fe.png)